lib/elements/CheckedSelect.js

```diff
@@ -174,6 +151,51 @@ class CheckedSelect extends React.Component {
                 </div>
             );
         });
+    };
+
+    renderMenu ({
+        focusedOption,
+        onFocus,
+        onSelect,
+        optionClassName,
+        optionComponent,
+        options,
+        valueArray,
+        valueKey
+    }) {
+
+        const props = this.props;
+        const filteredOptions = options.hasOwnProperty('length') ? options.length : 0;
```

Comment:
I think I found a small bug: this also counts options that have the `disabled:` property set to a truthy value, even though those will not be added if the button is clicked. I don't know if disabled options occur in the main use case right now, but to avoid surprises it's probably useful to `.filter()` or `.reduce()` or something.